classes/Image.php

```diff
@@ -726,7 +726,7 @@ public function getImgPath()
             return false;
         }
 
-        $path = $this->getImgFolder().$this->id;
+        $path = _PS_PROD_IMG_DIR_.$this->getImgFolder().$this->id;
```

Comment:
This changes the behavior of a public method, which used to return a relative path and now returns an absolute path.

Unfortunately, we cannot merge such a change until the next major version, because it could break other code relying on the previous behavior.